### PR TITLE
feat(bridge): relay burn event into 9c network

### PR DESCRIPTION
**Pull request by moreal**
This pull request continues #1 from #4. It implements to relay wNCG's `Burn` event to 9c network through GraphQL API.

### CODE REVIEW

File: bridge/src/ncg-transfer.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            query,
+            variables: {
+                recipient: address,
+                amount: amount.toString(),
+                txNonce: nextTxNonce,
+            }});
+
+        console.log(response);
+        console.log(response.data);
+        console.log(response.data.errors);
```
Comment (longfin) [inline bridge/src/ncg-transfer.ts:39]: Are these logs still needed?

Comment (moreal) [reply bridge/src/ncg-transfer.ts:39]: No, I used it for debugging, it will remove them from commits. Thanks 🙏🏻

Comment (moreal) [reply bridge/src/ncg-transfer.ts]: I applied it in new commits. Could you review them again?

**Comment by moreal** [comment]
@planetarium/9c-backend Could you review this pull request? 🙏🏻

File: bridge/src/ncg-transfer.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+    private async getNextTxNonce(address: string): Promise<number> {
+        const query = "query GetNextTxNonce($address: Address!) { nextTxNonce(address: $address) } ";
+        const response = await this.graphqlRequest({
+            operationName: "GetNextTxNonce",
+            query,
+            variables: { address, }
+        });
+
+        console.log(response);
```
Comment (longfin) [inline bridge/src/ncg-transfer.ts:48]: Still, log here 😅

**Review by longfin — APPROVED** [review]

**Review by area363 — APPROVED** [review]

File: bridge/src/ncg-transfer.ts (hunk 2 above)
Comment (moreal) [reply bridge/src/ncg-transfer.ts]: I removed also the log.

**Pull request merged**